Documents/customAnimal.md

```diff
@@ -0,0 +1,55 @@
+# How To Make A Custom Animal
+
+  
+
+## In this example we will be configuring a dragon animal.
+
+  
+
+### Where To Configure:
+
+The first thing you need to know is all configurations of the animal are done in console.
+
+  
+
+### Naming The Animal:
+
+The second thing you have to do is give the animal a name. The name of the animal does not effect any part of the program, so feel free to name it whatever you like!
+
+For Example,
+```console
+Animal: dragon
+```
+
+  
+
+### Setting Up Speed:
+
+Thirdly, what you need to do is input the speed of the animal, the strength of the animal, and the weight of the animal all on one line separated by a space. For example,
+
+```console
+Input speed, strength, weight: 4 17 160
+```
+
+if you want to find out how to increase or decrease speed, please refer to speedAlgorithm.md
```

Comment:
It is customary to link to reference docs... you can get the link to `speedAlgorithm.md` by right clicking on that file in GitHub and saving the link address.